pytorch/imagerecognition/cifar10-resnet20-all-reduce-scaling/main.py

```diff
@@ -60,14 +57,18 @@ def train_loop(
     rank = dist.get_rank()
     world_size = dist.get_world_size()
 
+    # LR = 0.1 / 256 / sample
```

Comment:
is this commented code needed?

Reply:
Yes, its just to explain where the formula comes from for the LR